test/e2e/tls/tls_test.go

```diff
@@ -18,14 +18,24 @@ import (
 var testRuntime *e2e.TestRuntime
 var pool *x509.CertPool
 
+var minTLSVersions = map[string]uint16{
+	"1.0": tls.VersionTLS10,
+	"1.1": tls.VersionTLS11,
+	"1.2": tls.VersionTLS12,
+	"1.3": tls.VersionTLS13,
+}
+
 // print error to stderr, exit 1
 func fatal(err interface{}) {
 	fmt.Fprintf(os.Stderr, "%s\n", err)
 	os.Exit(1)
 }
 
 func TestMain(m *testing.M) {
+	minTLSVersion := flag.String("--min-TLS-Version", "1.2", "minimum TLS Version")
```

Comment:
💡 I woudn't have thought that his works, since we define the flag as all-lowercase here: https://github.com/open-policy-agent/opa/pull/3517/files#diff-8146f8148ccbf6711d65f532f6ab9a7c8dfbdc3960c7ffc974f17d6d224dd349R183

To keep things consistent, we could use the lowercase variant here, too -- what do you think?

Reply:
sure I will make it all lowercase